tracer/tracer.go

```diff
@@ -180,20 +191,30 @@ func main() {
 		log.Fatalf("failed creating client: %v", err)
 	}
 	defer c.Close()
+`
+	var wfFn string
+	if tr.fnStruct != "" {
+		source += `
+        var fnStruct *fnpkg.` + tr.fnStruct
```

Comment:
Same assumption is made here, but here we would need to remove any leading and trailing parentheses to account for a pointer struct.